source/_components/switch.fritzbox_wifi.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "AVM FRITZBOX (Guest) WIFI Switch"
+description: "Instructions on how to integrate your AVM FRITZBOX (Guest) WIFI as switches into Home Assistant."
+date: 2018-12-23 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Switch
+ha_release:
+ha_iot_class: "Local Polling"
+---
+{% FRITZBOX WiFi Switch %}
```

Comment:
```suggestion
{% linkable_title FRITZBOX WiFi Switch %}
```

The line was correct only missing a name as title 😉

Reply:
😂🙈